frontend/src/main-components/CreateCardContent.js

```diff
@@ -1,22 +1,131 @@
-import React from "react";
+import React, { Component } from "react";
 import styled from "@emotion/styled";
+import css from "./CreateCardContent.css";
+import { motion } from "framer-motion";
+import FrontCard from "../flashcards/FlashcardFrontPreview";
+import BackCard from "../flashcards/FlashcardBackPreview";
+import { getTranslation } from "../sub-components/translate";
+import LangaugeScroll from "../sub-components/languageScroll";
```

Comment:
Space after imports.

Reply:
Ping?

Reply:
![Screenshot 2020-08-14 at 3 24 38 PM - Display 2](https://user-images.githubusercontent.com/33457643/90297158-775cce00-de42-11ea-8665-fe79c09ae7ab.png)
I have no idea why VScode adds this "fake" space but it took me a while to notice.